components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -584,12 +587,15 @@ internal sealed class SubscriptionWrapper : ISubscription {
                                     if (StatusCode.IsNotGood(results[i].StatusCode)){
                                         _logger.Warning("Set monitoring for item '{item}' in subscription "
                                             + "'{subscription}'/'{sessionId}' failed with '{status}'.",
-                                            itemsToChange[i].StartNodeId, 
+                                            itemsToChange[i].StartNodeId,
                                             Id,
                                             Connection.CreateConnectionId(),
                                             results[i].StatusCode);
+                                        changeList[i].Template.MonitoringMode = Publisher.Models.MonitoringMode.Disabled;
```

Comment:
Again, all ok, but I wonder whether null is appropriate monitoring mode for "failure case"?